src/Microsoft.AspNet.Http.Abstractions/HttpResponse.cs

```diff
@@ -16,34 +19,90 @@ public abstract class HttpResponse
             return Task.FromResult(0);
         };
 
+        /// <summary>
+        /// Gets the <see cref="HttpContext"/> for this request.
+        /// </summary>
         public abstract HttpContext HttpContext { get; }
 
+        /// <summary>
+        /// Gets or sets the HTTP response code.
+        /// </summary>
         public abstract int StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets the response headers.
+        /// </summary>
         public abstract IHeaderDictionary Headers { get; }
 
+        /// <summary>
+        /// Gets the response body <see cref="Stream"/>.
+        /// </summary>
         public abstract Stream Body { get; set; }
 
+        /// <summary>
+        /// Gets or sets the length of the response. If specified, this value will be used to set the value of the
```

Comment:
Use the same description as the ContentType property. This just gets/sets the header.